<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1402</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tactile Board Configuration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>290</height>
         </size>
        </property>
        <property name="title">
         <string>Tactile Board</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QComboBox" name="cbSerialPort"/>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="btnCalibration">
           <property name="text">
            <string>Calibrate</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="btnConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="btnStop">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="btnStart">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QCheckBox" name="chCalibration">
           <property name="text">
            <string>Use Calibration</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string>COM</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="chRecording">
           <property name="text">
            <string>Recording</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tactile Sensors</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="GraphicsLayoutWidget" name="pltTS0"/>
        </item>
        <item>
         <widget class="GraphicsLayoutWidget" name="pltTS1"/>
        </item>
        <item>
         <widget class="GraphicsLayoutWidget" name="pltTS2"/>
        </item>
        <item>
         <widget class="GraphicsLayoutWidget" name="pltTS3"/>
        </item>
        <item>
         <widget class="GraphicsLayoutWidget" name="pltTS4"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Plot Type</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QRadioButton" name="rbGray">
             <property name="text">
              <string>Grayscale</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rbColor">
             <property name="text">
              <string>Colormap</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="rbHeat">
             <property name="text">
              <string>Heatmap</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Sensitivity</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QRadioButton" name="rbLow">
             <property name="text">
              <string>Low</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rbMedium">
             <property name="text">
              <string>Medium</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="rbHigh">
             <property name="text">
              <string>High</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QRadioButton" name="rbDefault">
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Tactile Patch Curve</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="0">
          <widget class="QComboBox" name="cbTactilePatch">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="PlotWidget" name="pltTactilePatch"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Tactile Patch</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>horizontalLayoutWidget</zorder>
        <zorder>horizontalLayoutWidget</zorder>
        <zorder>cbTactilePatch</zorder>
        <zorder>label_3</zorder>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1402</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
